src/threading.c

```diff
@@ -240,9 +207,9 @@ void jl_init_threading(void)
 
 void jl_start_threads(void)
 {
-    char *cp;
+    char *cp, mask[UV_CPU_SETSIZE];
```

Comment:
ah sorry, my mistake - was in a docker container (ubuntu 14.04 x86_64) with a slightly unusual configuration that didn't have libuv rebuilding properly, nevermind